lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -0,0 +1,329 @@
+"""
+Run these tests @ Devstack:
+    paver test_system -s lms --fasttest --verbose --test_id=lms/djangoapps/course_structure_api
+"""
+# pylint: disable=missing-docstring,invalid-name,maybe-no-member,attribute-defined-outside-init
+from datetime import datetime
+
+from django.core.urlresolvers import reverse
+from django.test.utils import override_settings
+from oauth2_provider.tests.factories import AccessTokenFactory, ClientFactory
+from xmodule.modulestore.tests.django_utils import TEST_DATA_MOCK_MODULESTORE, ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+
+from courseware.tests.factories import GlobalStaffFactory, StaffFactory
+
+
+TEST_SERVER_HOST = 'http://testserver'
+
+
+class AuthMixin(object):
```

Comment:
This is not used anywhere else, so this functionality could just be moved into CourseViewTestMixin.setup